networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,83 @@ These release notes track the development of the Network Observability Operator
+(link:https://issues.redhat.com/browse/NETOBSERV-1283[*NETOBSERV-1283*])
+
+* Previously, in the Network Observability Operator details from *Operators* -> *Installed Operators*, the `FlowCollector` *Status* field might have reported incorrect information about the state of the deployment. The status field now shows the proper conditions with improved messages. The history of events is kept, ordered by event date. (link:https://issues.redhat.com/browse/NETOBSERV-1224[*NETOBSERV-1224*])
+
+* Previously, during spikes of network traffic load, certain eBPF pods were OOM-killed and go into `CrashLoopBackOff` state with the 429 error: 'Per stream rate limit exceeded (limit:xMB/sec) while attempting to ingest for stream'. Now, you can xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-troubleshooting-loki-tenant-rate-limit_network-observability-troubleshooting[configure the `LokiStack` custom resource `spec.limits.global.ingestion`] with the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications restart the pods in your cluster and stop the 429 rate-limit error. (link:https://issues.redhat.com/browse/NETOBSERV-975[*NETOBSERV-975*]) 
+
+* Previously when `processor.metrics.tls` was set to `PROVIDED` the `insecureSkipVerify` option value was forced to be `true`. Now you can set `insecureSkipVerify` to `true` or `false`, and provide a CA certificate if needed.  (link:https://issues.redhat.com/browse/NETOBSERV-1087[NETOBSERV-1087])
+
+[id="network-observability-operator-1.4.0-known-issues"]
+=== Known issue
+* Since the 1.2.0 release of the Network Observability Operator, using Loki Operator 5.6, a Loki certificate change periodically affects the `flowlogs-pipeline` pods and results in dropped flows rather than flows written to Loki. The problem self-corrects after some time, but it still causes temporary flow data loss during the Loki certificate change. This issue has only been observed in large-scale environments of 120 nodes or greater. (link:https://issues.redhat.com/browse/NETOBSERV-980[*NETOBSERV-980*])
+
+* Currently, larger DNS packets require the eBPF agent to look for DNS header outside of the 1st skb segment. new ebpf agent helper function need to implemented to support it. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/NETOBSERV-1304[*NETOBSERV-1304*])
+
+* Currently, DNS over TCP packets require the eBPF agent to look for DNS header outside of the 1st skb segment. new ebpf agent helper function need to implemented to support it. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/NETOBSERV-1245[*NETOBSERV-1245*])
```

Comment:
For these two, you might clarify that this is only affecting the DNS tracking feature, ie. when `spec.agent.ebpf.features` includes `DNSTracking` - so that users not using it aren't worried